gateway/src/management_switch.rs

```diff
@@ -48,7 +49,8 @@ pub struct SwitchConfig {
     #[serde(default = "default_udp_listen_port")]
     pub udp_listen_port: u16,
     pub local_ignition_controller_interface: String,
-    pub rpc_max_attempts: usize,
+    pub rpc_max_attempts_general: usize,
+    pub rpc_max_attempts_reset: usize,
```

Comment:
it occurred to me that the config file might look a little nicer if this was laid out like
```toml
[rpc_max_attempts]
general = 3
reset = 10
```
but, on the other hand, since nobody is really writing these config files save for the ones in the repo, it really doesn't matter that much...